dss/util/aws/logging.py

```diff
@@ -0,0 +1,56 @@
+import time
+
+import boto3
+import botocore.exceptions
+
+
+def log_message(log_group_name: str, log_stream_name: str, message: str):
+    """Logs a message to cloudwatch."""
+
+    logs_client = boto3.client("logs")
+
+    # An astute reader might notice that the exception handling is a bit wonky.  This is because exception classes for
+    # boto are dynamically created, and we cannot actually catch them. :(
```

Comment:
You can access these exceptions via `boto3.client("logs").exceptions.ResourceAlreadyExistsException` and `botocore.errorfactory.InvalidSequenceTokenException` (and their identities are the same across multiple client instances).

There are some services (like EC2) that don't expose their exceptions in the AWS API service model, but logs does.

https://github.com/boto/botocore/pull/1005 :)